docs/08-developer-guide/04-project/02-release-guide.md

```diff
@@ -212,14 +256,11 @@ git tag -s "ozone-$VERSION-RC$RC"
 ```
 
 :::tip
-If the command fails on MacOS, you may need to do the following additional steps:
-
-- Install a program to prompt you for your GPG key passphrase (example using homebrew):
-
-  ```bash
-  brew install pinentry-mac
-  ```
+If the command fails, you may need to do the following additional steps:
+:::
 
+<Tabs>
+<TabItem value="mac" label="Mac">
 - Tell git to use this program for signing:
```

Comment:
The `git config` part is platform-independent, so tabs should be added only for GnuPG settings.